lib/taskana-core-test/src/test/java/acceptance/user/UserServiceAccTest.java

```diff
@@ -3,39 +3,79 @@
 
-import acceptance.AbstractAccTest;
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.stream.Stream;
 import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.DynamicTest;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestFactory;
-import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.api.function.ThrowingConsumer;
 
 import pro.taskana.common.api.exceptions.InvalidArgumentException;
 import pro.taskana.common.api.exceptions.MismatchedRoleException;
-import pro.taskana.common.test.security.JaasExtension;
-import pro.taskana.common.test.security.WithAccessId;
+import pro.taskana.testapi.TaskanaInject;
+import pro.taskana.testapi.TaskanaIntegrationTest;
+import pro.taskana.testapi.security.WithAccessId;
 import pro.taskana.user.api.UserService;
 import pro.taskana.user.api.exceptions.UserAlreadyExistException;
 import pro.taskana.user.api.exceptions.UserNotFoundException;
 import pro.taskana.user.api.models.User;
 
 /** Acceptance test which tests the functionality of the UserService. */
-@ExtendWith(JaasExtension.class)
-class UserServiceAccTest extends AbstractAccTest {
-  private static final UserService USER_SERVICE = taskanaEngine.getUserService();
+@TaskanaIntegrationTest
+class UserServiceAccTest {
+
+  @TaskanaInject UserService userService;
+
+  protected User createExampleUser(String id) {
+    User user = userService.newUser();
+    user.setId(id);
+    user.setFirstName("Hans");
+    user.setLastName("Georg");
+    user.setFullName("Georg, Hans");
+    user.setLongName("Georg, Hans - (user-10-20)");
+    user.setEmail("hans.georg@web.com");
+    user.setPhone("1234");
+    user.setMobilePhone("01574275632");
+    user.setOrgLevel4("level4");
+    user.setOrgLevel3("level3");
+    user.setOrgLevel2("level2");
+    user.setOrgLevel1("level1");
+    user.setData("ab");
+
+    return user;
+  }
+
+  @WithAccessId(user = "admin")
+  @BeforeAll
+  void setup() throws Exception {
+    User user_teamlead = userService.newUser();
+    user_teamlead.setId("teamlead-1");
+    user_teamlead.setFirstName("Titus");
+    user_teamlead.setLastName("Toll");
+    user_teamlead.setFullName("Titus Toll");
+    user_teamlead.setLongName("Toll, Titus - (teamlead-1)");
+    user_teamlead.setEmail("titus.toll@web.de");
+    user_teamlead.setPhone("040-2951854");
+    user_teamlead.setMobilePhone("015637683197");
+    user_teamlead.setOrgLevel4("Novatec");
+    user_teamlead.setOrgLevel3("BPM");
+    user_teamlead.setOrgLevel2("Human Workflow");
+    user_teamlead.setOrgLevel1("TASKANA");
+    user_teamlead.setData("xy");
+    userService.createUser(user_teamlead);
+  }
 
   @WithAccessId(user = "user-1-2")
   @Test
   void should_ReturnUserWithAllFields_When_IdExisting() throws Exception {
-    User user = USER_SERVICE.getUser("teamlead-1");
+    User user = userService.getUser("teamlead-1");
```

Comment:
As I only migrated the test to the test-api and some test are using "teamlead-1" (_should_ThrowUserAlreadyExistException_When_TryingToCreateUserWithExistingId_, _should_ReturnUserWithAllFields_When_IdExisting_)

Should I just rewrite them to use "user-1-1"?

Reply:
Or any user, yes :)

Reply:
Alright, I will also rename the user id from "user-1-1" to "testuser1", so that it's easier to differ between test-data-users and the users which are used by the test-api for access control.